develop/tutorials/articles/110-portlets/03-spring-mvc/00-spring-mvc-intro.markdown

```diff
@@ -43,9 +43,90 @@ points on why that works in @product-ver@:
 You'll still need to provide the Liferay descriptor files `liferay-display.xml`
 and `liferay-portlet.xml`, and you'll need a `portlet.xml` descriptor.
 
-Develop a Spring MVC portlet WAR file with the appropriate descriptor files, and
-let the auto-deploy process and Liferay's WAB generator take care of converting
-your project to a Liferay-ready WAB.
+Develop a Spring MVC portlet WAR file with the appropriate descriptor files. 
+
+Import class packages your portlet's descriptor files reference by adding the
+packages to an `Import-Package` header in the
+`liferay-plugin-package.properties` file. 
+
+Here's an example `Import-Package` header:
+
+    Import-Package:\
+        org.springframework.beans.factory.xml,\
+        org.springframework.context.config,\
+        org.springframework.security.config,\
+        org.springframework.web.servlet.config
```

Comment:
@holatuwol Good catch. I included the classes by mistake (copied them from the import statements).